exercises/nucleotide-count/canonical-data.json

```diff
@@ -1,12 +1,25 @@
 {
   "exercise": "nucleotide-count",
-  "version": "1.0.0",
+  "comments": [
+    "Given a string of nucleotides, the count of each nucleotide is returned.",
+    "",
+    "It is the track's responsibility to add further language specific tests like:",
+    "input error handling (null/undefined parameters etc.)",
+    "referential transparency (i.e. evaluating, a function/method gives the same value for same arguments every time.)",
+    "",
+    "'error' and 'null' should be treated according to the languages' specifics and possibilities."
```

Comment:
since there is no more `null` that I see (I could have missed), the part about `null` wouldn't be necessary